### DISCUSSION TITLE
Drop support to Symfony 4.4

### DISCUSSION THREAD

**Discussion by llaville** [composer-unused/composer-unused#633] [Ideas]
Because Symfony 4.4 reached End-Of-Life since a long time now : see https://symfony.com/releases/4.4

**Comment by icanhazstring** [comment]
Drop it like its hot 🔥😉

**Reply by llaville** [reply]
On my way ... `symbol-parser` already fixed

**Reply by llaville** [reply]
@icanhazstring  Need a new version published of `symbol-parser` (without SF 4.4 support) to raise composer constraint and avoid conflicts.

**Reply by icanhazstring** [reply]
Will do asap

**Reply by llaville** [reply]
FYI: 

I've still two Symfony dependencies with version 4.4
```
symfony/filesystem                 v4.4.9             Symfony Filesystem Component
symfony/finder                     v4.4.11            Symfony Finder Component
```

Due to constraint
```
composer-unused/symbol-parser 0.2.2  requires  symfony/finder (^4.4 || ^5.3 || ^6.0 || ^7.0)
```

**Reply by icanhazstring** [reply]
Drop em both

**Reply by icanhazstring** [reply]
symbol-parser 0.2.5 tagged

**Reply by llaville** [reply]
You misunderstood me : the constraints depend on version of symbol-parser : and with current version 0.2.2 or even 0.2.4 we have these constraints up !

**Reply by icanhazstring** [reply]
Ah nevermind. Should be gone with the new version then 👍

**Reply by llaville** [reply]
PR #634 running
